docs/shared/conf.py

```diff
@@ -63,8 +64,8 @@ def add_base(paths):
 master_doc = 'index'
 
 # General information about the project.
-project = u'edX'
-copyright = u'2013, EdX Doc Team'
+project = 'edX'
+copyright = '{year}, EdX Doc Team'.format(year=datetime.datetime.now().year)
```

Comment:
eeek! Ok, these should all be roughly the same, with the year and "edX Inc." as the copyright holder.

Somehow, even with all three of these files including three different copyright holders, the published guide is getting and applying a single footer and it seems to be the one in docs/en_us/platform_api/source/conf.py (see http://edx.readthedocs.io/projects/edx-platform-api/en/latest/index.html).
That one is the best of the bunch!